emails/source-code-download.html

```diff
@@ -0,0 +1,238 @@
+                            role="presentation"
+                            style="mso-table-lspace: 0pt; mso-table-rspace: 0pt; word-break: break-word;">
+                            <tbody>
+                              <tr>
+                                <td class="pad"
+                                  style="padding-bottom:10px;padding-left:10px;padding-right:10px;padding-top:40px;">
+                                  <div
+                                    style="color:#101112;direction:ltr;font-family:Arial, Helvetica, sans-serif;font-size:16px;font-weight:400;letter-spacing:0px;line-height:120%;text-align:left;mso-line-height-alt:19.2px;">
+                                    <p style="margin: 0;">Forminer Team</p>
+                                  </div>
+                                </td>
+                              </tr>
+                            </tbody>
+                          </table>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <table class="row row-2" align="center" width="100%" border="0" cellpadding="0" cellspacing="0"
+            role="presentation" style="mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #ffffff;">
+            <tbody>
+              <tr>
+                <td>
+                  <table class="row-content stack" align="center" border="0" cellpadding="0" cellspacing="0"
+                    role="presentation"
+                    style="mso-table-lspace: 0pt; mso-table-rspace: 0pt; color: #000; width: 600px; margin: 0 auto;"
+                    width="600">
+                    <tbody>
+                      <tr>
+                        <td class="column column-1" width="100%"
+                          style="mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-weight: 400; text-align: left; padding-bottom: 5px; padding-top: 5px; vertical-align: top; border-top: 0px; border-right: 0px; border-bottom: 0px; border-left: 0px;">
+                          <table class="empty_block block-1" width="100%" border="0" cellpadding="0" cellspacing="0"
+                            role="presentation" style="mso-table-lspace: 0pt; mso-table-rspace: 0pt;">
+                            <tbody>
+                              <tr>
+                                <td class="pad">
+                                  <div></div>
+                                </td>
+                              </tr>
+                            </tbody>
+                          </table>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </td>
+      </tr>
+    </tbody>
+  </table><!-- End -->
+</body>
+
+</html>
```

Comment:
No empty line EOF

Reply:
It is default formatting for html. I will leave it as it is.